numpy/core/src/multiarray/ctors.c

```diff
@@ -1101,8 +1099,8 @@ PyArray_NewFromDescr_int(
          * (a.data) doesn't work as it should.
          * Could probably just allocate a few bytes here. -- Chuck
          */
-        if (is_empty) {
-            nbytes = descr->elsize;
+        if (nbytes == 0) {
+            nbytes = descr->elsize ? descr->elsize : 1;
```

Comment:
I guess this should be split of as well? Since it is not refcount related, just due to partial test suit runs, it blew up (I guess because of the caching).

Reply:
I don't care that much, but if you want to shoot off a quick CR fixing this one, I'll approve it